tests/Install-Scripts.Test/Assets/InstallationScriptTestsWithWindowsLineEndings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sdk": {
+    "version": "1.0.0-beta.19463.3",
+    "rollForward": "latestFeature"
+  }
+}
```

Comment:
Depending on the configuration, `git` command may change the line endings. It is highly likely that this file will have `\n` line endings when checked-out to a CI machine on Linux.

I suggest that we add a `.gitattributes` file next to this json file and mark it as "always crlf" by:
```
InstallationScripttestsWithWindowsLineEndings.json text eol=crlf
```